release/pyinstaller/cli_hook.py

```diff
@@ -8,12 +8,11 @@
     # catch pyinstaller bug
     # subprocess.CalledProcessError: Command '['/Applications/alphaDIA.app/Contents/Frameworks/alphadia', '-sS', '-c', 'import platform; print(platform.mac_ver()[0])']' returned non-zero exit status 2.
     # [36481] Failed to execute script 'cli_hook' due to unhandled exception!
-    if len(sys.argv) >= 3:
-        if (sys.argv[1] == "-sS") & (sys.argv[2] == "-c"):
-            import platform
+    if len(sys.argv) >= 3 and (sys.argv[1] == "-sS") & (sys.argv[2] == "-c"):
```

Comment:
what is the binary operator order with python?

Reply:
simplified it by replacing "&" with "and"